Tasks/AzureCLI/task.json

```diff
@@ -56,6 +54,18 @@
       "helpMarkDown": "Select the Azure Classic subscription for the deployment"
     },
     {
+      "name": "scriptType",
```

Comment:
We shouldn't have this input.  We can figure it out based on extension (and document that.  .cmd or .bat or .sh